annotation-processor/src/main/java/org/fulib/fx/FulibFxProcessor.java

```diff
@@ -107,11 +110,14 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
     }
 
     private void checkOverrides(TypeElement typeElement) {
+        Map<TypeElement, List<ExecutableElement>> eventMethods = helper.streamAllMethods(typeElement)
+            .filter(this::isEventElement)
+            .collect(groupingBy(method -> (TypeElement) processingEnv.getTypeUtils().asElement(method.getEnclosingElement().asType())));
```

Comment:
Sorry I was not clear about this. I'd rather use the method name as the key, as that is faster to look up.
Also, you seem to have a double conversion here (enclosing element -> type -> element)